src/storage/saver/filesystem.rs

```diff
@@ -0,0 +1,105 @@
+use super::Saver;
+use async_std::fs;
+use async_std::prelude::*;
+use failure::Error;
+use futures::future::BoxFuture;
+use futures::FutureExt;
+use futures::TryFutureExt;
+use log::info;
+use std::env;
+use std::path::PathBuf;
+use std::sync::Arc;
+use uuid::Uuid;
+
+#[cfg(feature = "local-fs")]
```

Comment:
We don't need this since it's only included in `mod.rs` if the feature is enabled.